docs/source/usage/workflows/memoryPerDevice.rst

```diff
@@ -0,0 +1,115 @@
+.. _usage-workflows-memoryPerDevice:
+
+Calculating the memory requirement per device
```

Comment:
### Nouns in Titels

My new book! :)

Reply:
WTH, I thought I already fixed that.
Sorry that you had to point it out again. :/